code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -1258,4 +1258,13 @@ GLOBAL_LIST_EMPTY(roundstart_station_closets)
 		return
 	return ..()
 
+/obj/structure/closet/secure_closet/hitby(atom/movable/hit_by, skipcatch, hitpush, blocked, datum/thrownthing/throwingdatum)
+	if(!istype(hit_by, /obj/item/spear))
+		return ..()
+	// We have to manually tweak throwforce for now
+	var/obj/item/spear = hit_by
+	spear.throwforce *= 2
+	. = ..()
```

Comment:
This is really silly and we shouldn't be doing it this way, we should just add a way to intercept the damage and modify it from there.

Reply:
What a compelling idea #90809 If only someone smart and handsome #90809 worked on a system like that #90809

Reply:
Wouldn't it be better to wait for that PR to get merged first then

Reply:
Either it gets merged and I update this pr or this pr gets merged and I update it